4_3-javaScript_algorithms/03.js

```diff
@@ -0,0 +1,18 @@
+function invertedTriangle(n) {
+    if (n >1 && Number.isInteger(n)) {
+        let espace = "";
+        let asterisk = "";
+        for (let i = 1;i <= n;i+=1) {
+            espace = "";
+            for (var j = i;j <= n-1;j+=1) {
```

Comment:
porque vc usou "var" nessa linha?

Reply:
Era pra ter sido `let` mesmo, é que em um dos milhares de testes que fiz pra conseguir fazer funcionar, eu tava usando o `j` fora do for, mas não deu certo e esqueci de alterar pra var haha